__tests__/pages/apply/[pid].test.js

```diff
@@ -18,8 +29,15 @@ process.env.NEW_MANDATORY_QUESTION_JOURNEY_ENABLED = 'false';
 
 describe('getServerSideProps', () => {
   it('should return a redirect object with the expected destination when new mandatory question feature flag is off ', async () => {
+    //this mocks getAdvertSchemeVersion()
+    axios.get.mockImplementation(() =>
+      Promise.resolve({
+        data: { schemeVersion: 2, internalApplication: true },
+      }),
+    );
```

Comment:
Minor point but we may as well shorthand these (https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue)

```suggestion
axios.get.mockResolvedValue({
  data: { schemeVersion: 2, internalApplication: true },
});
```